src/ArmTemplates/Common/Constants/GlobalConstants.cs

```diff
@@ -3,7 +3,7 @@
     public static class GlobalConstants
     {
         public static string CommandStartDateTime = "";
-        public const string AppShortName = "apimtemplate";
+        public const string AppShortName = "ArmTemplates";
```

Comment:
Where is this used? Are you sure this is not a breaking change?

Reply:
reverted